proto/identity/associations/association.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package xmtp.identity.associations;
+
+import "identity/associations/signature.proto";
+
+option go_package = "github.com/xmtp/proto/v3/go/identity/associations";
+option java_package = "org.xmtp.proto.identity.associations";
+
+// The first entry of any XID log, which is considered valid iff the XID can be
+// deterministically derived from the address and nonce.
+// The recovery address defaults to the initial_associated_address unless
+// there is a subsequent ChangeRecoveryAddress in the log.
```

Comment:
I still can't get my head around how unsigned XID creation that's associated with a wallet will work. We can chat tomorrow. Maybe I'm missing something.

Reply:
Updated the XID creation to allow for unsigned creation with a nonce of 0, as well as signed creation with other nonces

Reply:
Updated the XID creation to require no independent signature, but must be batched with an AddAssociation.